packages/graphql-modules/src/application/context.ts

```diff
@@ -1,3 +1,4 @@
+import { AsyncLocalStorage } from 'async_hooks';
```

Comment:
This is Node-specific. Are we ok with that?

Reply:
Good point, but, we used async_hooks before in #2461. Also, that one closed #1697 which should be re-opened now as is.

However, I'll change this to a dynamic import so that we remove the node requirement.

Reply:
Done.